chainer/gradient_check.py

```diff
@@ -269,8 +272,9 @@ def check_backward(func, x_data, y_grad, params=(),
         no_grads = [x.dtype.kind != 'f' for x in xs]
     else:
         if len(no_grads) != len(xs):
-            raise ValueError(
-                'Length of no_grads param and xs should be same.')
+            msg = ('Length of no_grads param and xs should be same.\n'
+                   'Actual: {0} != {1}'.format(len(no_grads), len(xs)))
+            raise ValueError(msg)
```

Comment:
You don't have to assign a temporary variable here.

Reply:
Thanks, I removed the redundant variable.